x-pack/platform/plugins/shared/streams/server/routes/streams/significant_events/route.ts

```diff
@@ -233,29 +228,30 @@ const generateSignificantEventsRoute = createServerRoute({
       throw badRequest('Streams are not enabled');
     }
 
-    const generatedSignificantEventDefinitions = await generateSignificantEventDefinitions(
-      {
-        name: params.path.name,
-        connectorId: params.query.connectorId,
-        currentDate: params.query.currentDate,
-        shortLookback: params.query.shortLookback,
-        longLookback: params.query.longLookback,
-      },
-      {
-        inferenceClient,
-        esClient: createTracedEsClient({
-          client: scopedClusterClient.asCurrentUser,
+    return fromRxjs(
+      generateSignificantEventDefinitions(
```

Comment:
💬 change this service to return an Observable which get updated when the sequential requests are done so we don't timeout: https://github.com/elastic/streams-program/issues/437

Reply:
this can be closed then right?